src/styles/notificationIcons.scss

```diff
@@ -61,9 +61,9 @@
   width: var(--ds-size-200, $ds-size-200);
 
   [auro-icon] {
+    --ds-auro-icon-size: var(--ds-size-300, $ds-size-300);
```

Comment:
the size should be `$ds-size-200`

Reply:
Yes it should be. But I thought is was wrong because it was breaking out of bounds when set to that side. Figured out there is a underlying issue with the svg implementation. I am setting up a meeting to talk about this.

Reply:
<img width="1253" alt="image" src="https://github.com/user-attachments/assets/ccee2ddc-f95a-4785-a117-0dd74a5f555d">

2 issues found during testing these icon on password input.
1. The eye icons get cut off on the sides. 
2. The size token does not get linked properly

Reply:
This is all fixed with the latest push